app/assets/stylesheets/sections/deploy.scss

```diff
@@ -0,0 +1,55 @@
+$dropbox-color: #1081de;
+$zip-color:     #a58463;
+
+.nav-upload {
+  .tab-description {
+    margin-left: 10px;
+
+    .title {
+      font-size: 14px;
+      font-weight: bold;
+    }
+  }
+
+  .fa {
+    font-size: 42px;
+  }
+
+  li {
+    &.active {
+      a, a:hover {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line